### PR TITLE
Added info on creating alert rule in Prometheus

### PR DESCRIPTION
**Pull request by AedinC**

### CODE REVIEW

**Comment by AedinC** [comment]
HTML preview here: http://file.ams.redhat.com/aecollin/JK-7886/README.html

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 1]
```diff
@@ -250,8 +250,33 @@ spec:
         name: additional-scrape-configs
         key: kafka-federate.yaml
 ----
+
 ====
 
+=== Configuring Prometheus alerts for Kafka instance limits
+
+If you use the Prometheus Operator in your monitoring environment, you can create a PrometheusRule custom resource with alerts defined for the https://access.redhat.com/articles/5979061[capacity of a Kafka instance]. You can then apply that rule to the cluster that you are federating the metrics to. For more information about configuring alerting rules, see https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/alerting.md/[Alerting] in the Prometheus Operator documentation.
```
Comment (bhardesty) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:258]:
```suggestion
If you use the Prometheus Operator in your monitoring environment, you can create a `PrometheusRule` custom resource with alerts defined for the https://access.redhat.com/articles/5979061[capacity of a Kafka instance]. You can then apply that rule to the cluster that you are federating the metrics to. For more information about configuring alerting rules, see https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/alerting.md/[Alerting] in the Prometheus Operator documentation.
```

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 2]
```diff
@@ -250,8 +250,33 @@ spec:
...
+
+.Example PrometheusRule custom resource for a Kafka broker storage limit alert
```
Comment (bhardesty) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:260]:
```suggestion
.Example `PrometheusRule` custom resource for a Kafka broker storage limit alert
```

File: docs/kafka/metrics-monitoring-kafka/README.adoc [hunk 3]
```diff
@@ -250,8 +250,33 @@ spec:
         name: additional-scrape-configs
         key: kafka-federate.yaml
 ----
+
 ====
 
+=== Configuring Prometheus alerts for Kafka instance limits
+
+If you use the Prometheus Operator in your monitoring environment, you can create a PrometheusRule custom resource with alerts defined for the https://access.redhat.com/articles/5979061[capacity of a Kafka instance]. You can then apply that rule to the cluster that you are federating the metrics to. For more information about configuring alerting rules, see https://github.com/prometheus-operator/prometheus-operator/blob/main/Documentation/user-guides/alerting.md/[Alerting] in the Prometheus Operator documentation.
+
```
Comment (bhardesty) [inline docs/kafka/metrics-monitoring-kafka/README.adoc:259]: Is this a procedure? If so, it should have the ```.Procedure``` heading and at least one step.

**Review by bhardesty — COMMENTED** [review]
Looking at the full document, it looks like this is structured as a procedure within a procedure. It should probably just be a regular procedure, on the same level as "Viewing metrics..." and "Configuring metrics...".

**Comment by AedinC** [comment]
> Looking at the full document, it looks like this is structured as a procedure within a procedure. It should probably just be a regular procedure, on the same level as "Viewing metrics..." and "Configuring metrics...".

Thanks Ben. I will rewrite it as a procedure.

**Review by bhardesty — APPROVED** [review]

**Pull request merged**